src/components/MenuItems/MenuItems.tsx

```diff
@@ -2,14 +2,15 @@ import './MenuItems.scss';
 import { ProductCard } from 'components/ProductCard/ProductCard';
 import { CategoryDto } from 'dtos/CategoryDto';
 import { ProductDto } from 'dtos/ProductDto';
-import { ReactElement } from 'react';
+import { Dispatch, ReactElement, SetStateAction } from 'react';
 
 interface Props {
   category: CategoryDto;
   products: ProductDto[];
+  dispatch: React.Dispatch<any>;
```

Comment:
we should probably avoid using `any`'s as eslint complains about it

Reply:
done